frontend/src/pages/ActivityReport/Pages/components/NextStepsRepeater.css

```diff
@@ -0,0 +1,11 @@
+
+.usa-form .ttahub-next-steps-repeater .usa-textarea {
+  /* Keep within bounds of navigator form with or without delete icon */
+  max-width: 460px;
+}
```

Comment:
not sure, is this actually doing anything? I removed it locally and didn't see any real effect, but maybe I'm not aware of what problem its solving

Reply:
If you add two step boxes. Then remove one of the steps. The text box width changes. This keeps it consistent.